bad.css

```diff
@@ -0,0 +1,4 @@
+#such-id {
+    display: block;
+    padding: 0px auto;
```

Comment:
Expected end of value but found 'auto'.
Values of 0 shouldn't have units specified.